qpm.json

```diff
@@ -9,13 +9,14 @@
     "additionalData": {
       "overrideSoName": "libcustom-types.so",
       "soLink": "https://github.com/sc2ad/Il2CppQuestTypePatching/releases/download/v0.1.0/libtypepatching.so",
-      "debugSoLink": "https://github.com/sc2ad/Il2CppQuestTypePatching/releases/download/v0.1.0/libtypepatching_debug.so"
+      "debugSoLink": "https://github.com/sc2ad/Il2CppQuestTypePatching/releases/download/v0.1.0/libtypepatching_debug.so",
```

Comment:
While u are here can u just get rid of these? They don't mean anything and are redundant